src/functions.h

```diff
@@ -344,6 +344,12 @@ void load_SouthernHawkesBay_v21p12(global_model_parameters *GLOBAL_MODEL_PARAMET
 void load_Motu_Bay1_v22p3(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
 void load_Motu_Bay2_v22p3(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
 void load_Motu_Bay3_v22p3(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
+void load_Omaio_Bay1_v22p5(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
+void load_Omaio_Bay2_v22p5(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
+void load_Omaio_Bay3_v22p5(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
+void load_Motu_Bay2_v22p5(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
+void load_Motu_Bay3_v22p5(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
+void load_TolagaBay_v22p5(global_model_parameters *GLOBAL_MODEL_PARAMETERS, int basinNum);
```

Comment:
Unfortunately the suggested change I made earlier needs this to be changed to "Tolaga_Bay_v22p5" now.